easybuild/toolchains/gcccore.py

```diff
@@ -37,3 +37,4 @@ class GCCcore(Gcc):
     # Replace the default compiler module name with our own
     COMPILER_MODULE_NAME = [NAME]
     SUBTOOLCHAIN = DUMMY_TOOLCHAIN_NAME
+    OPTIONAL = True
```

Comment:
@bartoldeman Can you clarify why `GCCcore` is optional?

It's the base of every toolchain, or am I overlooking something?

Reply:
it's not the base of the older toolchains with GCC 4.8 / intel 2015.2 and older.

Reply:
Right, I overlooked that...

Please clarify with a comment above:

```python
# GCCcore is only guaranteed to be present in recent toolchains
# for old versions of some toolchains (GCC, intel) it is not part of the hierarchy and hence optional